Plugins/FormatSwift/Plugin.swift

```diff
@@ -20,12 +20,15 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
     // specifying the targets selected in the plugin dialog.
     let inputTargets = argumentExtractor.extractOption(named: "target")
 
-    var inputPaths: [String]
+    // Only lint the files passed to `--files`
+    let inputFiles = argumentExtractor.extractOption(named: "files")
```

Comment:
This supports both files and directories, so maybe we should name the option `--paths` instead. Thoughts?